website/docs/zh/api/modules/module-methods.mdx

```diff
@@ -160,6 +161,14 @@ import(
 
 指定新 chunk 的名称。
 
+##### webpackFetchPriority
+
+<ApiMeta addedVersion="1.0.0" />
+
+- **类型：**: `"low" | "high" | "auto"`
+
+为指定的动态导入设置 [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority)。也可以通过使用 `module.parser.javascript.dynamicImportFetchPriority` 选项为所有动态导入设置全局默认值。
```

Comment:
```suggestion
为指定的动态导入设置 [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority)。也可以通过使用 [module.parser.javascript.dynamicImportFetchPriority](/config/module#moduleparserjavascriptdynamicimportfetchpriority) 选项为所有动态导入设置全局默认值。
```

Reply:
do it with module configuration document